airbnb/src/components/Calendar/CalendarHead.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const DAY = ['일', '월', '화', '수', '목', '금', '토'];
+
+function CalendarHead(props) {
+  const { year, month, setMonth, position } = props;
+
+  return (
+    <Form>
+      <Nav>
+        {position === 'leftBtn' && (
+          <Btn onClick={() => setMonth(month - 1)} position={position}>
+            &lt;
+          </Btn>
+        )}
+        <Year>
+          {year}년 {month}월
+        </Year>
+        {position === 'rightBtn' && (
+          <Btn onClick={() => setMonth(month + 1)} position={position}>
+            &gt;
+          </Btn>
+        )}
+      </Nav>
+      <Days>
+        {DAY.map((day) => {
+          return <Day key={day}>{day}</Day>;
+        })}
+      </Days>
+    </Form>
+  );
+}
+
+const Form = styled.section`
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  width: 400px;
+  margin-left: 10px;
+`;
+
+const Nav = styled.nav`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 10px;
+`;
+
+const Year = styled.div`
+  font-size: ${({ theme }) => theme.fontSizes.xl};
+  font-weight: bold;
+`;
+
+const Days = styled.div`
```

Comment:
div보다는 tag 문맥에 맞게 사용할 수 있는 부분을 더 적용해보시면 좋을 것 같아요 
ul-li 는 같이 오면 좋겠고, main, section, article tag등도 적용해볼 부분이 있는지 살펴보면 좋을 것 같습니다.
https://developer.mozilla.org/en-US/docs/Glossary/Semantics
```suggestion
const Days = styled.ul`
```